internal/exporter/prometheus/collector/power_collector.go

```diff
@@ -77,13 +96,60 @@ func (c *PowerCollector) updateDescriptors() {
 				nil,
 			)
 		}
+
+		// process metric descriptors
+		if _, exists := c.processJoulesDescriptors[zoneName]; !exists {
+			c.processJoulesDescriptors[zoneName] = prometheus.NewDesc(
+				prometheus.BuildFQName(namespace, processRAPL, zoneName+"_joules_total"),
+				"Energy consumption in joules for RAPL zone "+zoneName+" by process",
+				[]string{"pid", "comm", "exe", "container_id"},
+				nil,
+			)
+		}
+
+		if _, exists := c.processWattsDescriptors[zoneName]; !exists {
+			c.processWattsDescriptors[zoneName] = prometheus.NewDesc(
+				prometheus.BuildFQName(namespace, processRAPL, zoneName+"_watts"),
+				"Power consumption in watts for RAPL zone "+zoneName+" by process",
+				[]string{"pid", "comm", "exe", "container_id"},
+				nil,
+			)
+		}
+
+		// container metric descriptors
+		if _, exists := c.containerJoulesDescriptors[zoneName]; !exists {
+			c.containerJoulesDescriptors[zoneName] = prometheus.NewDesc(
+				prometheus.BuildFQName(namespace, containerRAPL, zoneName+"_joules_total"),
+				"Energy consumption in joules for RAPL zone "+zoneName+" by container",
+				[]string{"id", "name", "runtime", "namespace", "pod"},
```

Comment:
can we update the Help string to mention what each label means?

```suggestion
				"Energy consumption in joules for RAPL zone "+zoneName+" by container.  Labels: id = container id, name: container name, runtime: container runtime, namespace: kubernetes namespace (if available), pod: kubernetes pod name (if available)",
				[]string{"id", "name", "runtime", "namespace", "pod"},
```
same for other descriptors.

"namespace" is kubernetes namespace, it should be clear from metric description.